### ISSUE TITLE
Light chain gene identity required in Notebook 12.02

### ISSUE THREAD

**Issue by OLaprevote** [RosettaCommons/PyRosetta.notebooks#57]
In the third [antibody design notebook](https://github.com/RosettaCommons/PyRosetta.notebooks/blob/master/notebooks/12.02-RosettaAntibodyDesign-RAbD.ipynb), the AntibodyDesignMover `rabd2` crashes when applied, giving the error message:
```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-7-2dc5bb52b04a> in <module>
     10 rabd2.set_light_chain("kappa")
     11 if not os.getenv("DEBUG"):
---> 12     rabd2.apply(pose)

RuntimeError: 

File: /home/benchmark/rosetta/source/src/protocols/antibody/database/AntibodyDatabaseManager.cc:367
[ ERROR ] UtilityExitException
ERROR: Antibody Design requires the identity of the light chain gene. Please set it using the -light_chain option.  PyIgClassify can be used to determine the light chain gene.
```
XmlObjects calling the AntibodyDesignMover work fine, only the python AntibodyDesignMover seems to crash. Looking up `PyIgClassify` in pyrosetta API reference doesn't give any result. Setting `-light_chain kappa` during `pyrosetta.init` fixes this, so I fear the problem comes from PyRosetta.

Using PyRosetta version `2020.39+release.9ee477a`. The required version is 2020.38 in [`environment.yml`](https://github.com/RosettaCommons/PyRosetta.notebooks/blob/master/notebooks/environment.yml), so it would be worth checking with this one and a more recent PyRosetta release.

**Comment by jadolfbr** [comment]
Thanks for reporting this!  Fixed in PR #61

**Comment by jadolfbr** [comment]
This will actually be fixed via C++ code in a Rosetta merge today.
